XSOAR Mirroring Integration (#24189)
demisto/content: Packs/XSOARmirroring/Integrations/XSOARmirroring/README.md

Allows mirroring of XSOAR incidents between different Cortex XSOAR tenants.

This integration was integrated and tested with version 6.0 of XSOAR

> Mirrored incidents include the playbook ID.
> Note that the receiving side will attempt to run a playbook with a matching ID, if it exists locally.

Reviewer: ```suggestion
Mirrored incidents include the playbook ID. The receiving side will attempt to run a playbook with a matching ID, if one exists locally.
```
